powerlibs/django/restless/contrib/endpoints/base.py

```diff
@@ -86,3 +86,33 @@ class SoftDeletableListEndpointMixin:
     def get_query_set(self, request, *args, **kwargs):
         queryset = super().get_query_set(request, *args, **kwargs)
         return queryset.filter(deleted=False)
+
+
+class SieveListEndpoint:
```

Comment:
I don't think you should create a class for that. The behaviour of getting rid of `deleted` field should be implemented on `SoftDeletableEndpoint`. The same for `created_by` on `OwnedEndpoint`.